source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,170 @@
+.. _glossary:
+
+=========
+Glossary
+=========
+
+Machine learning terms
+======================
+
+.. glossary::
+    :sorted:
+
+    Categorical feature
+        A :term:`feature` with a discrete domain without ordering between values.
+
+        Synonyms: nominal feature.
```

Comment:
here we specify synonyms in one style and somewhere else in another style?

Reply:
I specify synonyms consistently across this document. @outoftardis, is it ok to indicate synonyms as I do? Or we have some other approved/used style.

Reply:
we don't have any other style, yours is ok, but I suggest these changes: 
- remove the full stop
- make the word **Synonyms** bold
- use **Synonym** if there is only one

The same goes for examples/example.

Reply:
Done